src/vivarium/framework/results/interface.py

```diff
@@ -145,6 +150,7 @@ def register_binned_stratification(
         ------
         None
         """
+        # TODO: implement excluded_categories like in `register_stratification`
```

Comment:
Will this be dealt with in a subsequent PR?

Reply:
@rmudambi  I wasn't sure how much of a priority this even is. It wasn't immediately clear to me how to handle excluded categories but then I'm also aware that we don't actually have an example of using `register_binned_stratification`.

Reply:
I added this to the tests